client/components/upgrades/gsuite/gsuite-upsell-card/index.jsx

```diff
@@ -85,7 +85,7 @@ const GSuiteUpsellCard = ( {
 			<CompactCard>
 				<header className="gsuite-upsell-card__header">
 					<h2 className="gsuite-upsell-card__title">
-						{ translate( 'Add Professional email from G Suite by Google Cloud to %(domain)s', {
+						{ translate( 'Add professional email from G Suite by Google Cloud to %(domain)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.